clang/test/Driver/ppc32-fcomplex-ppc-gnu-abi.c

```diff
@@ -0,0 +1,15 @@
+// RUN: not %clang %s --target=x86_64 -fcomplex-ppc-gnu-abi 2>&1 \
+// RUN:     | FileCheck %s -check-prefix=X86_64
+// X86_64: error: unsupported option '-fcomplex-ppc-gnu-abi' for target 'x86_64'
+
+// RUN: not %clang %s --target=ppc64 -fcomplex-ppc-gnu-abi 2>&1 \
```

Comment:
I think two test scenarios is enough , 

`if (!TC.getTriple().isPPC32() || !TC.getTriple().isOSBinFormatELF()) {`

one for `!TC.getTriple().isPPC32()` , other for `!TC.getTriple().isOSBinFormatELF()`